openedx/core/djangoapps/catalog/utils.py

```diff
@@ -210,6 +210,31 @@ def get_course_runs():
     return course_runs
 
 
+def get_course_runs_for_course(course_uuid):
+    catalog_integration = CatalogIntegration.current()
+
+    if catalog_integration.is_enabled():
+        try:
+            user = catalog_integration.get_service_user()
+        except ObjectDoesNotExist:
+            logger.error(
+                'Catalog service user with username [%s] does not exist. Course runs will not be retrieved.',
+                catalog_integration.service_username,
+            )
+            return []
+
+        api = create_catalog_api_client(user)
+        cache_key = '{base}.course'.format(base=catalog_integration.CACHE_KEY)
+        data = get_edx_api_data(
```

Comment:
As discussed with @staubina we need a ling term cache TTL value for these entires.

Reply:
oh and tests :)